### PR TITLE
Update to hive 2.1.1

### PR DESCRIPTION
**Pull request by wmiel**

### CODE REVIEW

File: examples/rspec-hive.yml.example [hunk 1]
```diff
@@ -3,4 +3,10 @@ hive:
   port: 10000
   host_shared_directory_path: /Users/Shared/tmp/spec-tmp-files
   docker_shared_directory_path: /tmp/spec-tmp-files
-  hive_version: 10
\ No newline at end of file
+  hive_version: 13
+  hive_options:
+    'hive.exec.dynamic.partition': 'true'
+    'hive.exec.dynamic.partition.mode': 'nonstrict'
+    'hive.exec.max.dynamic.partitions.pernodexi': '100000'
```
Comment (mkrawc) [inline examples/rspec-hive.yml.example]: I don't think this setting exist and I think it has never existed in Hive, it should be `hive.exec.max.dynamic.partitions.pernode`

File: docker/Dockerfile [hunk 2]
```diff
@@ -17,31 +17,45 @@ ENV SHARED_FOLDER /tmp/spec-tmp-files/
...
 
-RUN wget https://www.apache.org/dist/hadoop/core/hadoop-1.2.1/hadoop-1.2.1-bin.tar.gz -O $HADOOP_BIN/hadoop.tar.gz
-RUN wget https://archive.apache.org/dist/hive/hive-0.11.0/hive-0.11.0-bin.tar.gz -O $HIVE_BIN/hive.tar.gz
+RUN wget http://www-eu.apache.org/dist/hadoop/common/hadoop-2.7.3/hadoop-2.7.3.tar.gz -O $HADOOP_BIN/hadoop.tar.gz
+RUN wget http://www-eu.apache.org/dist/hive/hive-2.1.1/apache-hive-2.1.1-bin.tar.gz -O $HIVE_BIN/hive.tar.gz
+# RUN wget http://ftp.ps.pl/pub/apache/tez/0.8.5/apache-tez-0.8.5-bin.tar.gz -O $TEZ_BIN/tez.tar.gz
 
 RUN tar -xzf $HADOOP_BIN/hadoop.tar.gz --strip-components=1 -C $HADOOP_HOME
 RUN tar -xzf $HIVE_BIN/hive.tar.gz --strip-components=1 -C $HIVE_HOME
+# RUN tar -xzf $TEZ_BIN/tez.tar.gz --strip-components=1 -C $TEZ_HOME
```
Comment (mkrawc) [inline docker/Dockerfile]: Remove the commented out line.

File: examples/rspec-hive.yml.example [hunk 3]
```diff
@@ -3,4 +3,10 @@ hive:
   port: 10000
   host_shared_directory_path: /Users/Shared/tmp/spec-tmp-files
   docker_shared_directory_path: /tmp/spec-tmp-files
-  hive_version: 10
\ No newline at end of file
+  hive_version: 13
```
Comment (mkrawc) [inline examples/rspec-hive.yml.example]: I couldn't make it working with hive_version 13. Does it work for you?

**Comment by madlew** [comment]
@mkrawc please ignore the existing dockerfile - it'l be overwritten once we merge our changes

**Comment by mkrawc** [comment]
@wmiel check all other occurances of `pernodexi`, it must have been a mistake long time ago, and I have no idea how it worked.

**Comment by wmiel** [comment]
@mkrawc Thanks for this `pernodexi`, good catch 👍

File: rspec-hive.gemspec [hunk 4]
```diff
@@ -7,7 +8,7 @@ Gem::Specification.new do |spec|
   spec.name          = 'rspec-hive'
   spec.version       = RSpec::Hive::VERSION
   spec.authors       = ['Wojtek Mielczarek', 'Mikołaj Nowak']
-  spec.email         = %w(wojtek.mielczarek@u2i.com mikolaj.nowak@u2i.com)
+  spec.email         = %w[wojtek.mielczarek@u2i.com mikolaj.nowak@u2i.com]
```
[the remark this answers is not in the document]
Comment (wmiel) [reply rspec-hive.gemspec]: Rubocop suggests that: `%i-literals should be delimited by [ and ]`

**Pull request merged**